### PR TITLE
Add configuration file for sensitive information

### PR DESCRIPTION
**Pull request by x0rz3q**

### CODE REVIEW

**Comment by x0rz3q** [comment]
To use it basically copy the example, then fill out the info and compile.

**Comment by x0rz3q** [comment]
That github actions change went better than expected.

File: b0tweb/Configuration_example.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace b0tweb
+{
+    class Configuration
+    {
+        public static string IRCPassword = "";
```
Comment (JurisMajors) [inline b0tweb/Configuration_example.cs]: missing docs ;p

Comment (x0rz3q) [reply b0tweb/Configuration_example.cs]: Nice catch

**Pull request merged**